Dockerfile

```diff
@@ -72,47 +62,34 @@ RUN apt-get update && \
 	python3-wheel \
 	${with_arm64_as:+binutils-aarch64-linux-gnu} \
 	${with_arm32_as:+binutils-arm-linux-gnueabi} \
-	${with_ppc_as:+binutils-powerpc64le-linux-gnu} && \
-	pip3 install meson ninja && \
-	cd /tmp && \
-	git clone -b "$RZ_PIPE_PY_VERSION" https://github.com/rizinorg/rz-pipe && \
-	pip3 install ./rz-pipe/python && \
-	cd rizin && \
-	meson --prefix=/usr /tmp/build && \
-	meson compile -C /tmp/build && \
-	meson install -C /tmp/build && \
-	rm -rf /tmp/build && \
-	rz-pm init && \
-	rz-pm -gi rz-ghidra && \
-	rm -rf $HOME/.local/share/rizin/rz-pm/git/rz-ghidra && \
-	pip3 uninstall -y meson ninja && \
-	apt-get remove --purge -y \
-	cmake \
-	cpp \
-	g++ \
-	pkg-config \
-	python3-pip \
-	python3-setuptools \
-	python3-wheel && \
-	apt-get autoremove --purge -y && \
-	apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+	${with_ppc_as:+binutils-powerpc64le-linux-gnu}
 
+RUN pip3 install meson ninja
+
+WORKDIR /tmp
+RUN git clone -b "$RZ_PIPE_PY_VERSION" https://github.com/rizinorg/rz-pipe
+RUN pip3 install --root=/tmp/rizin-install ./rz-pipe/python
+
+WORKDIR /tmp/rizin
+RUN meson --prefix=/usr /tmp/build
+RUN meson compile -C /tmp/build
+RUN meson install --destdir /tmp/rizin-install -C /tmp/build
+
+
+FROM debian:10
 ENV RZ_ARM64_AS=${with_arm64_as:+aarch64-linux-gnu-as}
 ENV RZ_ARM32_AS=${with_arm32_as:+arm-linux-gnueabi-as}
 ENV RZ_PPC_AS=${with_ppc_as:+powerpc64le-linux-gnu-as}
 
-# Create non-root user
 RUN useradd -m rizin
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates git python3
```

Comment:
Wouldn't you also need a C/C++ toolchain for `rz-pm` plugins?

Reply:
Yes but i wouldn't install it by default and leave that to the user or another layer of images based on this. To avoid big images. Otherwise i should also keep cmake, ninja, meson and other things